.circleci/main.yml

```diff
@@ -1304,6 +1304,7 @@ jobs:
             git commit -m "bump pachyderm version"
             git push origin release-${CIRCLE_TAG:1}
             PR=$(gh pr create --base ${parentBranch} --title "bump pach version for release ${CIRCLE_TAG:1}" --body "bump pach version" --reviewer pachydermbuildbot --repo pachyderm/haberdashery)
+            sleep 10 # give it a bit time for checks to attach
```

Comment:
As a heuristic, I normally put any `sleep` command in a ticket later to improve. Finding a better wait condition might be good here as a starter ticket for a newer devops too.